src/accumulator.rs

```diff
@@ -122,8 +126,8 @@ pub enum State<Candidate, Digest, Signature> {
 #[derive(Debug, Default)]
 #[cfg_attr(any(test, feature="codec"), derive(Encode, Decode))]
 struct VoteCounts {
-	prepared: usize,
-	committed: usize,
+	prepared: u64,
+	committed: u64,
```

Comment:
as a note: if we know it won't overflow u32 it is better to stick with u32 as simplearithmetic can convert from u32 easily.